File: src/main/java/Result.java
public class Result
{
    private String path, score, code, err, output;

    public Result (String path, String score, String code, String err, String output)
    {
        setPath(path);
        setCode(code);
        setScore(score);
        setErr(err);
        setOutput(output);
    }

    public String getPath() {
        return path;
    }

    public void setPath(String path) {
        this.path = path;
    }

    public String getScore() {
        return score;
    }

    public void setScore(String score) {
        this.score = score;
    }

    public String getCode() {
        return code;
    }

    public void setCode(String code) {
        this.code = code;
    }

    public String getErr() {
        return err;
    }

    public void setErr(String err) {
        this.err = err;
    }

    public String getOutput() {
        return output;
    }

    public void setOutput(String output) {
        this.output = output;
    }
}

File: src/main/java/Result.kt
data class Result (val path: String, val score:String, val code:String, val err:String, val output:String)
/*
If you want to add more attributes to the Result class, it's real easy. All you have to do is add another argument to
this constructor here, and it will automatically make getter and setter method's for it. The syntax is easy too!
It's just "val (name): (Type)" seperated by commas. "val" is short for value, name is whatever you want that attribute's
name to be, and the Type is any standard Java type. The only little oddity is that primitives use capital names in Kotlin.
some examples:
val x:Int, val y:Double, val z:String, val asdf:FileBrowser
You can use any java or Kotlin class, and they are interchangable. You can even turn a java class into a kotlin one
automatically if you want! For example, the kotlin version of the

 */

